stories/Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader.jsx

```diff
@@ -30,11 +30,11 @@ const GlobalHeader = ({
               data={languageswitcherData.languagedata}
               headerText={languageswitcherData.headerText}
             />
-            <button className='menu-icon'></button>
+            <button className='menu-icon'>mobile bar</button>
```

Comment:
@Ashish-srij Any specific reason this text is required? If not can you remove it?

Reply:
@shishirphadke blank space is creating a violation issue.

Reply:
Ok

Reply:
@shishirphadke Done.